### PR TITLE
[REF] split sale_margin_delivered into sale_margin_delivered and sale_margin_delivered_security

### PR DESCRIPTION
**Pull request by legalsylvain**

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]
Thank you @legalsylvain

File: sale_margin_delivered_security/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (chienandalu) [inline sale_margin_delivered_security/readme/CONTRIBUTORS.rst:1]: Put yourself into :)

**Review by chienandalu — APPROVED** [review]
Thanks. Tiny remark:

File: sale_margin_delivered_security/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (legalsylvain) [reply sale_margin_delivered_security/readme/CONTRIBUTORS.rst:1]: done !

**Pull request merged**